drivers/hmis/spec/requests/hmis/delete_file_spec.rb

```diff
@@ -0,0 +1,87 @@
+require 'rails_helper'
+require_relative 'login_and_permissions'
+require_relative '../../support/hmis_base_setup'
+
+RSpec.describe Hmis::GraphqlController, type: :request do
+  before(:all) do
+    cleanup_test_environment
+  end
+  after(:all) do
+    cleanup_test_environment
+  end
+
+  include_context 'hmis base setup'
+  include_context 'file upload setup'
+
+  let!(:e1) { create :hmis_hud_enrollment, data_source: ds1, project: p1, client: c1, relationship_to_ho_h: 1, household_id: '1', user: u1 }
+  let!(:f1) do
+    file = Hmis::File.new(
+      name: blob.filename,
+      client_id: c1.id,
+      enrollment_id: e1.id,
+      effective_date: Date.today,
+      expiration_date: Date.tomorrow,
+      user_id: hmis_user.id,
+      confidential: true,
+      visible_in_window: false,
+    )
+    file.tag_list.add([tag.id])
+    file.client_file.attach(blob)
+    file.save!
+
+    file
+  end
+
+  let(:mutation) do
+    <<~GRAPHQL
+      mutation DeleteClientFile($fileId: ID!) {
+        deleteClientFile(input: { fileId: $fileId }) {
+          client {
+            id
+            files {
+              nodes {
+                id
+                url
+                name
+                contentType
+                effectiveDate
+                expirationDate
+                confidential
+                createdAt
+                updatedAt
```

Comment:
I prefer using this helper function, so that as we add new fields to the `File` type they are automatically resolved by the tests
```suggestion
                #{scalar_fields(Types::HmisSchema::File)}
```